using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference.md

```diff
@@ -118,17 +133,20 @@ ACPCampaign.setLinkageFields({"linkageKey": "linkageValue"});
 
 ## Reset linkage fields
 
-Removes the previously stored linkage fields in the SDK and triggers the Campaign rules download again. Personalized messages that were previously stored in the cache are erased.
+This method clears cached rules from previous download before triggering a rule download request to the configured Campaign server. If the current SDK privacy status is not OPT_IN, no rules download happens.
```

Comment:
- Please add "the" between "clears" and "cached rules" and add "the" between "from" and "previous".

- Please replace "happens" with "occurs".